src/scss/components/_file-upload.scss

```diff
@@ -107,9 +107,9 @@ we don't have strict BEM naming here.
       align-items: center;
 
       &__file {
-        @include ellipsis;
+        overflow-wrap: break-word;
```

Comment:
fyi, `break-word` is deprecated: https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#syntax

You can do the same by using:
```
word-break: normal;
overflow-wrap: anywhere;
```